lib/src/external_module.cc

```diff
@@ -362,12 +355,13 @@ ActionOutcome ExternalModule::callNonBlockingAction(const ActionRequest& request
         { lth_exec::execution_options::merge_environment });  // options
 
     if (exec.exit_code == EXTERNAL_MODULE_FILE_ERROR_EC) {
-        LOG_DEBUG("The '%1% %2%' non-blocking task %3% failed to write its "
-                  "output on file%4%%5%",
-                  module_name, action_name, request.transactionId(),
-                  (exec.output.empty() ? "" : std::string("; output: ") + exec.output),
-                  (exec.output.empty() ? "" : std::string("; error: ") + exec.error));
-        throw Module::ProcessingError { "failed to write output to files" };
+        // This is unexpected. The outcome of the task will not be
+        // available for future transaction status requests
+        LOG_WARNING("The task process failed to write output on file for the %1%%2%%3%",
+                    request.prettyLabel(),
+                    (exec.output.empty() ? "" : "; stdout: " + exec.output),
+                    (exec.error.empty() ? "" : "; stderr: " + exec.error));
```

Comment:
I'd prefer always including the ```stdout: ``` & ```stderr: ``` even if they are empty - in which case I'd just say ```(empty)``` similarly as above.